WordPress/src/main/java/org/wordpress/android/util/config/LandingScreenRevampFeatureConfig.kt

```diff
@@ -9,4 +9,8 @@ import javax.inject.Inject
  */
 @FeatureInDevelopment
 class LandingScreenRevampFeatureConfig
-@Inject constructor(appConfig: AppConfig) : FeatureConfig(appConfig, BuildConfig.LANDING_SCREEN_REVAMP)
+@Inject constructor(appConfig: AppConfig) : FeatureConfig(appConfig, BuildConfig.LANDING_SCREEN_REVAMP) {
+    override fun isEnabled(): Boolean {
+        return super.isEnabled() || BuildConfig.IS_JETPACK_APP
```

Comment:
This s more of a nitpick suggestion and I don't expect that it affects performance at all. I would go with the `BuildConfig.IS_JETPACK_APP` first to avoid calling the `super` method for the Jetpack app.

Reply:
Good point, I'll make an update for both remarks (this & [above](https://github.com/wordpress-mobile/WordPress-Android/pull/17351#discussion_r999094860))

Thank you for reviewing @antonis 🙇